[RUM-sdk][NextJs] Fixes SSR for NextJS13+ (#20529)
DataDog/documentation: content/en/real_user_monitoring/guide/monitor-your-nextjs-app-with-rum.md

2. Click the **New Application** button.
3. Make sure JS is selected, then enter a name for your application and click **Create New RUM Application**. A `clientToken` and `applicationId` are automatically generated for your application.

> <div class="alert alert-info">When using `.env.local`, only variable prefixed with `NEXT_PUBLIC_` are included in the client bundle. See [environemt variables][9].</div>

Reviewer: ```suggestion
<div class="alert alert-info">When using `.env.local`, only variables prefixed with `NEXT_PUBLIC_` are included in the client bundle. See [environment variables][9].</div>
```

   If using NPM, you need to make a few small changes to the code snippet from the Datadog RUM UI before pasting it into either the root `layout.tsx` or custom `_app.tsx` file (Datadog supports both):

>    - If your application relies on the **newer** Next.js [App Router][1] (versions 13+), paste the snippet into the root [`layout.tsx`][2] file. Note that it will prevent server rendering, hence it is recommended to use the CDN version instead.

Reviewer: ```suggestion
   - If your application relies on the **newer** Next.js [App Router][1] (versions 13+), paste the snippet into the root [`layout.tsx`][2] file. The `use client` directive prevents server-side rendering, hence it is recommended to use the CDN version instead.
```
